test/e2e/eraser_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+type Name string
+
+const podTest Name = "pod-test"
+
+func TestPodOnNodesObjects(t *testing.T) {
+	podFeat := features.New("Test pod").
+		Setup(func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			// deploy eraser config
+			if err := deployEraserConfig(cfg.KubeconfigFile(), cfg.Namespace(), "examples", "eraser_v1alpha1_imagelist.yaml"); err != nil {
+				t.Error("Failed to deploy config", err)
+			}
+			pod := newPod(cfg.Namespace(), "pod-test")
```

Comment:
what is this pod for?

Reply:
what we want is for after deploying an imagelist for manager to schedule and run eraser in all the nodes then remove images from list from the nodes

Reply:
As I mentioned, this is only the framework. I'll add the test cases once we merged them. I thought it would be many changes at once.